pkg/factory/config.go

```diff
@@ -107,13 +109,30 @@ func (c *Configuration) validate() (bool, error) {
 
 type Sbi struct {
 	Scheme       string `yaml:"scheme" valid:"scheme"`
-	RegisterIPv4 string `yaml:"registerIPv4,omitempty" valid:"host,required"` // IP that is registered at NRF.
-	BindingIPv4  string `yaml:"bindingIPv4,omitempty" valid:"host,required"`  // IP used to run the server in the node.
-	Port         int    `yaml:"port,omitempty" valid:"port,required"`
+	RegisterIPv4 string `yaml:"registerIPv4,omitempty" valid:"host,optional"` // IP that is registered at NRF.
+	RegisterIPv6 string `yaml:"registerIPv6,omitempty" valid:"host,optional"` // IP that is registered at NRF.
```

Comment:
I thought we said to only have:
```go
	RegisterIPv4 string `yaml:"registerIPv4,omitempty" valid:"host,optional"` // IP that is registered at NRF.
	RegisterIP string `yaml:"registerIPv4,omitempty" valid:"host,optional"` // IP that is registered at NRF.
```
With RegisterIPv4 being the fallback if RegisterIP does not exist, and RegisterIP accepting both IPv6 and IPv6?